torch/_ops.py

```diff
@@ -99,39 +99,59 @@ def __getattr__(self, key):
         if key == '__file__':
             return 'torch.ops'
 
+        # ensure that query for dunder attributes that does not exist on
+        # opoverloadpacket but instead exists on the self._op object does not unnecessarily call
+        # `_get_operation_overload` (which is an expensive operation).
+        # This is done to prevent any potential slowdown. This list can be extended
+        # if there exists other attributes like `__name__` that only exist on self._op and not on the
+        # opoverloadpacket.
```

Comment:
nit: I think we can mention here that this is ok as aten ops cannot start with "__" ever.
Also since we are guaranteed that anything that starts with "__" isn't an op, not sure we need the "endswith" check?